lib/Dialect/FIRRTL/Transforms/LowerGroups.cpp

```diff
@@ -11,6 +11,7 @@
 
 #include "PassDetails.h"
 #include "circt/Dialect/FIRRTL/FIRRTLUtils.h"
+#include "circt/Dialect/FIRRTL/Namespace.h"
```

Comment:
Is this necessary?

Reply:
The file uses `CircuitNamespace`, so should include the header declaring it :+1:.